### DISCUSSION TITLE
v3.6.0 has incompatible changes?

### DISCUSSION THREAD

**Discussion by roubaeli** [pycontribs/jira#1796] [General]
It seems that v3.6.0 removed the createmeta methods (`createmeta_issuetypes` and `createmeta_fieldtypes`) and replaced them with two new methods (`project_issue_types` and `project_issue_fields`).

This is possibly a backward incompatible change that is not easily discovered by inspecting the release version number or by going through the release's change log. Deprecation warnings raised by the obsolete `createmeta` method also still point users to these two removed methods.

Is that maybe a communication problem?

**Comment by mdoar** [comment]
That would be a problem for at one large user I know of!
